src/fragments/gen2/quickstart/build-a-backend.mdx

```diff
@@ -98,33 +102,25 @@ const schema = a.schema({
     .authorization([a.allow.owner(), a.allow.public().to(['read'])]),
 });
 
-export type Schema = ClientSchema<typeof schema>;
-
-export const data = defineData({
-  schema,
-  authorizationModes: {
-    defaultAuthorizationMode: 'apiKey',
-    // API Key is used for a.allow.public() rules
-    apiKeyAuthorizationMode: {
-      expiresInDays: 30,
-    },
-  },
-});
-
 // ...
 ```
 
-Once you save your changes to the data model, your changes will be deployed in seconds within your cloud sandbox.
+Once you save your changes to the data model, they will be deployed in seconds to your cloud sandbox.
+
+
+The `Todo` data model is defined with authorization rules to allow the person who creates the Todo instance (the owner), to perform all actions on the data they own.  We are also allowing all page viewers, including unauthenticated users, to read data.
 
-Our Todo data model also has authorization set up. Our model has the `authorization` method chained to it, which you can add rules to. In our example, we are allowing an owner, or the person who creates the Todo instance, to perform all actions on the data they own. We are also allowing all page viewers, including unauthenticated users, to read data. These can be modified; for example, we could remove the `.to(['read'])` and allow all visitors to perform all actions on data. We could also add permissions for signed-in users or users who belong to user groups such as `Admin`. You can learn more about all options for authorization in the [customize your auth rules](/gen2/build-a-backend/data/customize-authz/) section of the docs.
+_Note: These authorization rules can be modified using a chain of methods as defined by default. For example, we could remove the `.to(['read'])` and allow all visitors to perform all actions on data or add permissions for signed-in users or users who belong to user groups such as `Admin`. You can learn more about all options for authorization in the [customize your auth rules](/gen2/build-a-backend/data/customize-authz/) section of the docs._
 
-Let's remove public access.
+<b>STEP 2</b> Remove public access by deleting the `a.allow.public().to(['read'])` authorization rule. You authorization rule will look like the code below:
 
 ```js
 .authorization([a.allow.owner()]),
 ```
 
-Then, you will see the `defineData` function, which has our schema and authorization configuration passed in as arguments. We have an `apiKey` set up to enable public access. Let's change our `defaultAuthorizationMode` to `userPool` instead so that the default is to use user authentication.
+Below the schema declaration, you will see the `defineData` function which receives our schema and authorization configuration as arguments.  The default configuration is for an `apiKey` to enable public access.
+
+<b>STEP 3</b> Update the `defaultAuthorizationMode` to `userPool` so that the default is to use user authentication.
 
 ```js
 export const data = defineData({
```

Comment:
Again here wonder if we should in the // ... so they quickly see it is abbreviate
```suggestion
// ...

export const data = defineData({
```